packages/web-components/fast-foundation/src/listbox/listbox.ts

```diff
@@ -333,7 +260,7 @@ export class Listbox extends FoundationElement {
 
         switch (key) {
             // Select the first available option
-            case "Home": {
+            case keyHome: {
                 if (!e.shiftKey) {
                     e.preventDefault();
```

Comment:
I think the calls to preventDefault here are unnecessary as that is already handled by the boolean return value of the function though the template binding.

Reply:
Since this base method is overridden in `Select` and `Combobox`, the returned value for this function only affects the template binding behavior when called for `ListboxElement`. Relying on the return value to handle `preventDefault` gets really confusing when working with supermethod overrides.

Reply:
Ok.  I wish we did not have different signatures for the template vs regular event listeners.